aws_vpc_policies/aws_security_group_unused_security_group.yml

```diff
@@ -0,0 +1,141 @@
+AnalysisType: policy
+Filename: aws_security_group_unused_security_group.py
+PolicyID: AWS.SecurityGroup.UnusedSecurityGroup
+Enabled: true
```

Comment:
I would set this to `Enabled: false` by default, as this policy only works if a specific non-default deployment option is set.